docs/position.rst

```diff
@@ -3,7 +3,7 @@ position
 
 .. dfhack-tool::
     :summary: Report cursor and mouse position, along with other info.
-    :tags: adventure fort inspection map
+    :tags: adventure dfhack fort inspection map
```

Comment:
I updated the spreadsheet to reflect this: https://docs.google.com/spreadsheets/d/1hiDlo8M_bB_1jE-5HRs2RrrA_VZ4cRu9VXaTctX_nwk/edit?pli=1&gid=1774645373#gid=1774645373